cdxev/__main__.py

```diff
@@ -709,6 +711,49 @@ def create_build_public_bom_parser(
     return parser
 
 
+# noinspection PyUnresolvedReferences,PyProtectedMember
+def create_init_sbom_parser(
+    subparsers: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparsers.add_parser(
+        "init-sbom",
+        help=("Provides the first draft of an SBOM for manual completion."),
```

Comment:
Please also add the `usage`, like e.g. done for [set](https://github.com/Festo-se/cyclonedx-editor-validator/blob/5b836c5266c02619edd5f1a0d3a2f277b4434715/cdxev/__main__.py#L568), so that `target` will be shown in the help

Reply:
Done